lib/index.js

```diff
@@ -256,24 +267,12 @@ class HttpErrorBase extends Error {
     this.statusCode = res.status
     this.code = 'E' + res.status
     this.method = method
-    this.target = target
+    this.target = res.url
```

Comment:
how does `res.url` differ from `target` and if they don't, why do we still have `target`?

Reply:
I don't know, tbh. I think we can probably drop this.